ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -401,5 +539,6 @@ type component_opennebula = {
     'host_ovs'      ? boolean
     'host_hyp'      : string = 'kvm' with match (SELF, '^(kvm|xen)$')
     'tm_system_ds'  ? string with match(SELF, "^(shared|ssh|vmfs)$")
+    'v5_config'     ? boolean
```

Comment:
https://forum.opennebula.org/t/question-about-var-lib-one-remotes-version-file-and-one-versions/2699